compiler/aarch64/codegen/OMRLinkage.cpp

```diff
@@ -32,13 +33,338 @@
 #include "il/Node.hpp"
 #include "il/Node_inlines.hpp"
 #include "il/ParameterSymbol.hpp"
+#include "il/SymbolReference.hpp"
+#include "infra/IGNode.hpp"
+#include "infra/InterferenceGraph.hpp"
+
+typedef bool (*localSizeTestMethod) (uint32_t size);
+static bool isLocalSize4Byte(uint32_t size)
+   {
+   return (size == 4);
+   }
+static bool isLocalSize8Byte(uint32_t size)
+   {
+   return (size == 8);
+   }
+static bool isLocalSizeLargerThan8Byte(uint32_t size)
+   {
+   return (size > 8);
+   }
+
+void OMR::ARM64::Linkage::mapCompactedStack(TR::ResolvedMethodSymbol *method)
+   {
+   const TR::ARM64LinkageProperties& linkageProperties = getProperties();
+   ListIterator<TR::AutomaticSymbol> automaticIterator(&method->getAutomaticList());
+   TR::AutomaticSymbol *localCursor;
+   TR::CodeGenerator *cg = self()->cg();
+   TR::Compilation *comp = self()->comp();
+
+   TR::GCStackAtlas *atlas = cg->getStackAtlas();
+
+   {
```

Comment:
How about adding a comment like "// scope of the stack memory region starts here" ?

Reply:
I added a comment at the start of the block.